### ISSUE TITLE
Move to nursery/infra team oversight

### ISSUE THREAD

**Issue by aturon** [nrc/highfive#81]
The infra team is interested in helping maintain this piece of infrastructure, starting with introducing some debugging to figure out why some PRs are continuing to miss assignments. @aidanhs, in particular, has volunteered to help.

Are you open to moving this repo to the nursery?

We should explore the relationship to Servo's fork before making any move.

**Comment by nrc** [comment]
I'd be happy to move it. The harder thing is moving the hosting. It's currently running on my personal server, we'd need to find a infra-team maintained server to host it somewhere.

It is different enough from Servo's fork that merging them is very non-trivial. Definitely do-able, but not easy.

**Comment by aidanhs** [comment]
Yes, the hosting is mainly what I'd be looking at. In terms of timeline, I'll probably start working on it early December.

**Comment by nrc** [comment]
OK, I moved highfive to the nursery as a first step and have given the infra team commit access. However, until we move the hosting, I at least need CCing on PRs since I will need to upload the changes to the server.

**Comment by dwijnand** [comment]
Should this move to https://github.com/rust-ops next to https://github.com/rust-ops/homu?
